File: string/check-if-numbers-are-ascending-in-a-sentence/check-if-numbers-are-ascending-in-a-sentence.go
package checkifnumbersareascendinginasentence

import (
	"strconv"
	"strings"
)

func areNumbersAscending(s string) bool {
	tokens := strings.Fields(s)
	last := -1
	for _, t := range tokens {
		c, err := strconv.Atoi(t)
		if err != nil {
			continue
		}
		if c <= last {
			return false
		}
		last = c
	}
	return true
}

File: string/check-if-numbers-are-ascending-in-a-sentence/check-if-numbers-are-ascending-in-a-sentence.ts
function areNumbersAscending(s: string): boolean {
    const tokens = s.split(' ');
    let last = -1;

    for (let token of tokens) {
        let number = parseInt(token)
        if (isNaN(number)) {
            continue;
        }
        if (number <= last) {
            return false;
        }
        last = number;
    }

    return true;
};
